src/main/resources/plugin.yml

```diff
@@ -356,13 +356,77 @@ permissions:
     - mf.force.claim
     - mf.force.unclaim
     - mf.force.flag
+  mf.player:
+    description: Player permission
+    default: false
```

Comment:
Does this parent permission being false make the children permissions false too?

Reply:
I don't think so, but I can certainly test it tonight